docs/5-Integrations.md

```diff
@@ -12,18 +12,47 @@
 
 ## Drupal 7
 ### Using twig in D7
-- Twigshim module
-- How do I set up the variable mapping
-- Kalaponents + ¶ == component based workflow
-- If you make use of Kalaponents’ “Component” Paragraphs bundle, it takes care of mapping the basic Drupal fields to their template counterparts:
-	- title
-	- text
-	- image.src
-	- link.text
-	- link.url
-- If you need to make additional adjustments to the variables, implement hook_kalaponents_data_alter().
-- If you want to adjust the markup returned from Twig (e.g., to wrap the output in a section), implement hook_kalaponents_markup_alter().
-- When sending other theme output through twigshim_render() (e.g., other Paragraph bundles, blocks, node templates), you must set up the Twig template variables manually and pass them in to twigshim.
+
+[Twigshim](https://github.com/kalamuna/twigshim) is a small helper 
+[module](https://www.drupal.org/docs/7/extending-drupal-7/installing-contributed-modules) 
+for [Drupal 7](https://www.drupal.org/drupal-7.0) that pulls in
+[Symfony](https://symfony.com/)'s [Twig](http://twig.sensiolabs.org/) library 
+via [Composer](https://getcomposer.org/). It is an "API module", which means it 
+doesn't do anything on its own; it just exposes a `twigshim_render()` function 
+that accepts a Twig template file path and an array of template variables.
+Developers can use this function to bypass Drupal's standard render pipeline and 
```

Comment:
Splitting line breaks at 60/80 characters is a pain to maintain for documentation. While we could leave it for this, I request we avoid that going forwards.